language_service/src/hover/tests.rs

```diff
@@ -46,7 +46,7 @@ fn hover_callable_unit_types() {
         }
     "#},
         &expect![[r#"
-            "```qsharp\noperation Bar() : Unit\n```\nDoc comment\nwith multiple lines!\n"
+            "```qsharp\nTest\noperation Bar() : Unit\n```\nDoc comment\nwith multiple lines!\n"
```

Comment:
I just noticed you're using the `assert_debug_eq` function instead of the `assert_eq` function. The former gets the debug representation of the string, which adds unnecessary `"` and `\n` characters.... you don't really need the debug representation since you're just comparing strings. I suggest changing to `assert_eq` and updating all the expect strings, I suspect it'll look much prettier.

Reply:
That's a big improvement! Thanks!